### PR TITLE
Task/shank 223 survey resource

### PR DESCRIPTION
**Pull request by jhulbert-va**
[SHANK-223](https://vasdvp.atlassian.net/browse/SHANK-223)

* Table Creation sql script
* _**2**_ Survey Datamart Files

### CODE REVIEW

File: db/migration/V4_0__create_survey.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+-----------------|-----------------------------------------------------
+SurveyName       | The type of survey that was performed (i.e. Morse Fall Scale). 
+-----------------|-----------------------------------------------------
+CDWId            | The CDW identifier for the survey.
+-----------------|-----------------------------------------------------
+Sta3n            | The station identifier of where the survey was performed.
+-----------------|-----------------------------------------------------
+Survey           | The json formatted survey datamart resource payload.
+-----------------|-----------------------------------------------------
+ETLBacthId       | ETL process BatchId
```
Comment (dlennon-va) [inline db/migration/V4_0__create_survey.sql]: `Bacth`

File: db/migration/V4_0__create_survey.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                 | the record
+-----------------|-----------------------------------------------------
+
+========================================================================== */
+
+Create table [app].[Survey]
+(
+        [CDWId] varchar(50) not null,
+        [PatientFullICN] varchar(50) not null,
+        [DateUTC] datetime null,
```
Comment (dlennon-va) [inline db/migration/V4_0__create_survey.sql]: Don't see this in the description comment above, also should we make it match the field we are actually going to use? `surveySavedDateTime`

Comment (jhulbert-va) [reply db/migration/V4_0__create_survey.sql]: `surveySavedDateTime` is part of the datamart survey resource. I do agree that maybe `DateUTC` might not be the place for it though. I'd like to see what everyone else thinks about this one. @eclendenning-va @ilaflamme-va @bschofield-va

**Comment by jhulbert-va** [comment]
# DO NOT MERGE UNTIL MINIMART IS UPDATED

File: db/migration/V4_0__create_survey.sql (hunk 2 above)
Comment (ilaflamme-va) [reply db/migration/V4_0__create_survey.sql]: I do think that DateUTC is a bit ambiguous, but for the mock work I'm not particular.

**Pull request merged**